executor/distsql.go

```diff
@@ -824,14 +827,25 @@ func (w *tableWorker) executeTask(ctx context.Context, task *lookupTableTask) er
 		sort.Sort(task)
 	}
 
-	if w.isCheckOp && handleCnt != len(task.rows) {
-		obtainedHandlesMap := make(map[int64]struct{}, len(task.rows))
-		for _, row := range task.rows {
-			handle := row.GetInt64(w.handleIdx)
-			obtainedHandlesMap[handle] = struct{}{}
+	if handleCnt != len(task.rows) {
```

Comment:
We can use a variable to indicate if we need to check consistency when building the executor, it will simply the execution code.

Reply:
Inconsistent problem is a severe problem and this check has no performance side effect, so let it do this check every time may be better and safer.